pages/science/reactors/cvodes.rst

```diff
@@ -0,0 +1,244 @@
+    >>> gas.TPX = 1000.0, ct.one_atm, "H2:2,O2:1,N2:4" #set gas to an interesting state
+    >>> reac = ct.IdealGasReactor(gas)                 #create a reactor containing the gas
+    >>> sim = ct.ReactorNet([reac])                    #add the reactor to a new ReactorNet simulator
+    >>> gas()                #view the initial state of the mixture (state summary will be printed to console)
+    >>> sim.advance(1)       #advance the simulation to the specified absolute time, t = 1 sec
+    >>> gas()                #view the updated state of the mixture, reflecting properties at t = 1 sec
+
+For a more advanced example that adds inlets and outlets to the reactor, see Cantera's combustor example 
+(`Python <https://github.com/Cantera/cantera/blob/main/interfaces/cython/cantera/examples/reactors/combustor.py>`__ 
+| `C++ <https://github.com/Cantera/cantera/blob/main/samples/cxx/combustor/combustor.cpp>`__). Additional examples 
+can be found in the `Python Reactor Network Examples <https://cantera.org/examples/python/index.html#python-example-
+reactors>`__ section of the Cantera website.
+
+Step 2: ``advance()`` method called
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In any case, after properly configuring a reactor network and its components in Cantera, a call to the 
+``ReactorNet``'s ``advance()`` method can be used to predict the state of the network at a specified time. 
+The initial condition information is passed off to the `Integrator` when calling `advance()`.
+Transient physical and chemical interactions are simulated by integrating the network's system of ODE 
+governing equations through time, a process that's actually performed by an external `Integrator` object.
+
+Step 3: Information about current gas state provided to an `Integrator`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``Integrator`` class is Cantera's interface for ODE system integrators.
+
+``Integrator`` is a `polymorphic base class <http://www.cplusplus.com/doc/tutorial/polymorphism/>`__; it 
+defines a set of *virtual* functionalities that derived classes (the actual ODE system integrators) will 
+provide implementations for.
+
+**Integrator.h** creates a ``newIntegrator()``. Factory Method ``newIntegrator()`` creates and returns a 
+pointer to an ``Integrator`` instance of type ``itype``. The ``newIntegrator()`` instance will automatically 
+have an ``itype`` of ``CVODES``, which is installed with Cantera. The ``newIntegrator()`` will be stored as 
+variable ``m_integ``.
+
+Step 4: Communicate with CVODES using a wrapper function
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Because ``CVODES`` is written in C, the ``CVodesIntegrator`` C++ wrapper is used to access the solver.
+The ``CVodesIntegrator`` class is a C++ wrapper class for ``CVODES``. (`Documentation 
+<https://cantera.org/documentation/docs-2.4/doxygen/html/d9/d6b/classCantera_1_1CVodesIntegrator.html>`__)
```

Comment:
Let's definitely not link to the Cantera 2.4 documentation -- please use the version-independent way of doing this.